src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/ExportJobTaskTests.cs

```diff
@@ -1607,6 +1620,7 @@ public async Task GivenAnUnKnownExceptionThrowFromAnonymizer_WhenExecuted_ThenJo
                 _resourceDeserializer,
                 factory.CreateMockScope(),
                 Substitute.For<IMediator>(),
+                Substitute.For<IFhirRequestContextAccessor>(),
```

Comment:
There constructor looks like it is repeated a lot

Reply:
Yea, I can replace that with a method call. Most of them are identical.